openreferee_server/server.py

```diff
@@ -163,6 +163,17 @@ def get_event_info(event):
     return EventInfoSchema().dump(event)
 
 
+@app.route('/event/<identifier>/contributions/<contrib_id>/editing/<any(paper,slides,poster):file_type>',
```

Comment:
`editable_type`, not `file_type`

with a url like `/event/<identifier>/editable/<editable_type>/<contrib_id>`, using PUT would probably work fine as well.